### ISSUE TITLE
Missing options

### ISSUE THREAD

**Issue by stephenegriffin** [MicrosoftDocs/visualstudio-docs#4788]
Article suggests that warnings can be suppressed or work items created. There must be some pre-req for this to happen, because neither are in the right click menu for warnings in the Error List. Further, "the Actions link at the bottom of the warning" is not a thing I can locate anywhere - a warning is a single line entry in the Error list - where would the bottom be?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f4074bba-2193-a551-616f-54fa34d27192
* Version Independent ID: f01119e7-4ac0-0850-1842-452ef34e9a74
* Content: [Quickstart: Code Analysis for C/C++ - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/code-quality/quick-start-code-analysis-for-c-cpp?view=vs-2019#feedback)
* Content Source: [docs/code-quality/quick-start-code-analysis-for-c-cpp.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/code-quality/quick-start-code-analysis-for-c-cpp.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-code-analysis**
* GitHub Login: @mikeblome
* Microsoft Alias: **mblome**

**Comment by stephenegriffin** [comment]
Further, this sentence is unclear: "To analyze a specific warning, choose the title of the warning in the Error List." - again - errors in the Error List are single line entries - there is no expansion visible.

**Comment by stephenegriffin** [comment]
Clarifying - this is VS 2019 and a C++ project using the v142 toolset

**Comment by corob-msft** [comment]
@stephenegriffin 

Thanks for the report. After a bit of research, it appears warning suppression isn't an option in native C++, so I've removed that from the article. I've updated other parts to refer to the current UI behavior in the Error List window. Specifically, there's an expansion triangle in the first column of the Error List grid when more information is available. Choose it to see the extra info. There's no Action button, but the actions available are found in the right-click context menu that comes up for each warning. 

The fixes should appear within a day of this issue being closed.
